src/test/java/com/datadoghq/datadog_lambda_java/EnhancedMetricTest.java

```diff
@@ -23,7 +23,9 @@ public void makeTagsFromContext() {
         Assert.assertTrue(tags.containsKey("runtime"));
         Assert.assertTrue(tags.containsKey("resource"));
         Assert.assertTrue(tags.containsKey("executedversion"));
+        Assert.assertTrue(tags.containsKey("datadog_lambda_version"));
 
+        Assert.assertEquals("0.0.5", tags.get("datadog_lambda_version"));
```

Comment:
Will we need to update this test on every version?

Reply:
Yes, so I'll remove it to reduce hard-coding

Reply:
Maybe it would be better to assert that the datadog_lambda_version matches some version-y regex

Reply:
added regex that checks for x.x.x format